File: Fundamentals of C programming/step07/7.1/7.1.03.c
/*

Усовершенствовать программу, написанную на прошлом шаге: дополнительно на отдельной строке вывести количество напечатанных чисел.

Входные данные:
Два целых числа K,MK,M. При этом MM больше KK.

Выходные данные:
Натуральные числа в порядке возрастания, принадлежащие промежутку [K,M][K,M]. Числа нужно разделять одним пробелом.
На новой строке вывести количество выведенных натуральных чисел, а если натуральные числа отсутствуют -- ноль.

Уточнение:

На конце первой строки обязательно должен ставиться пробел.

Sample Input 1:

3 12

Sample Output 1:

3 4 5 6 7 8 9 10 11 12 
10

Sample Input 2:

-3 3

Sample Output 2:

1 2 3 
3

*/

#include <stdio.h>

int main(void) {

  int k,m,count=0;
  scanf("%d %d",&k,&m);
  
  for (int i = k; i <= m; i++){
	if (i>0) {
	count++;
    printf("%d ",i);
    }
  }

  printf("\n%d\n",count);

  return 0;
}

File: Diving into JavaScript: For Beginners/step07/7.01/7.1.03.js
/*

Создайте объект budget со свойствами income (доход) и expenses (расходы), значения которых вводятся пользователем и методом calculateProfit, 
который высчитывает прибыль по следующей формуле:

    Прибыль = доход - расходы

Если прибыль положительная, программа выводит значение прибыли. Если прибыль равна нулю, программа выводит сообщение о том, 
что пользователь отработал в ноль. Если прибыль отрицательна, программа выводит насколько пользователь ушел в минус. 

Используйте интерполяцию.

Sample Input 1:

15000
5000

Sample Output 1:

Ваша прибыль составляет 10000 рублей

Sample Input 2:

525200
525200

Sample Output 2:

Вы отработали в ноль

Sample Input 3:

5
130000

Sample Output 3:

Вы ушли в минус на 129995 рублей

*/

const budget = {
    calculateProfit () {
        return this.income - this.expenses
    }
};

budget.income = Number(prompt());
budget.expenses = Number(prompt());

if (budget.calculateProfit() > 0) {
    console.log(`Ваша прибыль составляет ${budget.calculateProfit()} рублей`);
} else if (budget.calculateProfit() < 0) {
    console.log(`Вы ушли в минус на ${-budget.calculateProfit()} рублей`);
} else if (budget.calculateProfit() === 0) {
    console.log('Вы отработали в ноль');
}
